buildSrc/src/main/kotlin/com/datadog/gradle/plugin/checklicenses/SPDXLicense.kt

```diff
@@ -6,6 +6,7 @@
 
 package com.datadog.gradle.plugin.checklicenses
 
+@Suppress("ktlint:enum-entry-name-case")
```

Comment:
Suppressing the check here as I dont think it is worth modifying all enum variables defined here...